lecture_2/05_offerfeed/internal/infrastructure/http/axilisOfferFeed.go

```diff
@@ -2,12 +2,20 @@ package http
 
 import (
 	"context"
+	"encoding/json"
+	"io/ioutil"
+	"log"
 	"net/http"
+	"strconv"
+	"strings"
+	"time"
 
 	"code-cadets-2021/lecture_2/05_offerfeed/internal/domain/models"
+	"github.com/pkg/errors"
 )
 
 const axilisFeedURL = "http://18.193.121.232/axilis-feed"
+const axilisFeedURL2 = "http://18.193.121.232/axilis-feed-2"
```

Comment:
And why should you structure the code differently?

Maybe in a year there will be a new company - InfinumFeeder, which wont expose their feed through HTTP server.
They could expose their feed over gRPC, or some other protocol.

Then, you would already be prepared:

you would create a new infinumFeed.go file which is an implementation of Feed interface
thats it, just plug it in your code

Reply:
Thank you for the input, I'll make the changes soon. My idea was to make the urls work in one feed and even if one fails, other can still provide data. But the separate feeds for each is a better way.

Reply:
> My idea was to make the urls work in one feed and even if one fails, other can still provide data

Good and correct idea. Implement it like that, but just with separate feeds. (Dont kill the app if one feed is misbehaving, you can still work with the other one).
Kill the app only if user presses Ctrl + C or if `Queue` fails to start.

Reply:
Sorry for a later update, but i still managed to make separate `feeds` and the `feedMerger.go`.
I thought it would be better if i made it as a separate `task` and to not put all the merging into `feedProcessorService`.
Thank you for all inputs regarding this PR.